cmake/define-test.cmake

```diff
@@ -103,4 +103,31 @@ function(define_test)
         ${WRAPPERS}
         partition-alloc)
     add_dependencies(check-ia2 ${MAIN})
+
+    # Find libc
+    list(APPEND LIBC_PATHS /lib/libc.so.6 /lib64/libc.so.6 /lib/x86_64-linux-gnu/libc.so.6)
+    foreach(CANDIDATE in ${LIBC_PATHS})
+        if (EXISTS ${CANDIDATE})
+            set(LIBC_PATH ${CANDIDATE})
+        endif()
+    endforeach()
+    if(NOT DEFINED LIBC_PATH)
+        string(REPLACE ";" "\t" PATHS "${LIBC_PATHS}")
+        message(FATAL_ERROR
+            "Could not find libc.so.6 in:\n${PATHS}\n")
+    endif()
+
+    # Generate libc with padded TLS segment
+    add_custom_command(
```

Comment:
Same comments as above.

Reply:
The same logic applies here--we need both because we want to create a target that behaves like a library and also associate the output files with the command.